devel/fetch_metrics/fetch_metrics.py

```diff
@@ -225,6 +238,18 @@ def output_results(comparison_list):
         print(f"Results:\n{json.dumps(comparison_list, indent=4)}")
 
 def main():
+    use_container = os.getenv('HITHER_USE_CONTAINER') in ['TRUE', '1']
```

Comment:
Decide whether to use container based on env variables. Maybe we'll want to add cli options in the future.